website/docs/install.md

```diff
@@ -32,7 +32,7 @@ Learn the basics before getting started, how to install, where is the location o
 Using `npm`
 
 ```bash
-npm install --location=global verdaccio@next-7
+npm install -g verdaccio@next-7
```

Comment:
🤔 I'm pretty sure the `--location` was quite new but I don't find it anymore at the docs.

Reply:
--location is for `npm config set` (or `npm set`).
Not for install 😊

Reply:
I did the modification for the other place. There is another commit in this pull request.